settlement/dymension/dymension.go

```diff
@@ -505,8 +506,11 @@ func (c *Client) GetBondedSequencers() ([]types.Sequencer, error) {
 	return sequencerList, nil
 }
 
-// CheckRotationInProgress implements settlement.ClientI.
-func (c *Client) CheckRotationInProgress() (*types.Sequencer, error) {
+// GetNextProposer returns the next proposer on the hub.
+// In case there is a current proposer but no next proposer, it returns nil.
+// in case there isnt current or next proposer, it returns an empty sequencer struct.
+// in case there is a next proposer, it returns the next proposer.
```

Comment:
```suggestion
// GetNextProposer returns the next proposer on the hub.
// In case the current proposer is the next proposer, it returns nil. 
// in case there is no next proposer, it returns an empty sequencer struct.
// in case there is a next proposer, it returns the next proposer.
```